Podcast/ActionSheetViewController.swift

```diff
@@ -30,12 +30,16 @@ class ActionSheetHeader {
     
 }
 
-protocol ActionSheetViewControllerDelegate: class {
+protocol ActionSheetViewControllerPlayerControlsDelegate: class {
     func didPressSegmentedControlForTrimSilence(selected: Bool)
     func didPressSegmentedControlForSavePreferences(selected: Bool)
 }
 
-class ActionSheetViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, ActionSheetPlayerControlsTableViewCellDelegate {
+protocol ActionSheetViewControllerRecastBlurbDelegate: class {
```

Comment:
Why make a second delegate for this? What about putting it all in one delegate (ActionSheetViewControllerDelegate) and having optional functions to implement